projects/storefrontapp/src/app/app.module.ts

```diff
@@ -43,7 +40,6 @@ if (!environment.production) {
 @NgModule({
   imports: [
     BrowserModule.withServerTransition({ appId: 'spartacus-app' }),
-    BrowserTransferStateModule,
```

Comment:
Is this change really related to this PR?

Reply:
No, it's not. I figured to do the cleanup here, since I'm testing both ssr and prerendering.
Do you want to remove it from this PR?